bin/install.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+export DB_PORT=27018 # Edit with the wanted port
```

Comment:
I'm not sure someone using the automagic installer will tune this; and I guess the variable is obvious enough to avoid the current comment.